sns_ik_lib/src/fosns_velocity_ik.cpp

```diff
@@ -93,12 +93,15 @@ Scalar FOSNSVelocityIK::getJointVelocity(VectorD *jointVelocity,
     }
   }
 
-  double nSatTot = 0.0;
-  for (int i = 0; i < n_tasks; i++)
-    nSatTot += nSat[i];
-  return nSatTot;
+//  double nSatTot = 0.0;
```

Comment:
There's a bunch of old code like this sitting around. We should clean it up in the general code review sweep.